app/services/description_generator.rb

```diff
@@ -146,5 +147,18 @@ def contributor(work_form_contributor)
       ]
     }
   end
+
+  sig { returns(T::Array[T::Hash[String, T::Array[T::Hash[String, String]]]]) }
+  def related_resource
+    work.related_links.map do |rel_link|
+      uri_hash = {
+        "uri": rel_link.url
```

Comment:
I think `"uri":` here should be `uri:` in line with the changes in https://github.com/sul-dlss/happy-heron/pull/407/files#diff-f21f5ecf9be9f56bd3c574ea7cb992626d29a9a41cc2926f9f643eadf3a15c55 Because these keys are not actually strings; they're symbols. Not sure why Sorbet is tricked by this!

Reply:
ah, great!  yeah, i thought the `"string":` construction for hash keys was a bit odd, but i figured i'd stay consistent.   mean to ask but forgot, happy to change.

Reply:
I wonder if some of this could move into a new `RelatedLink#to_h` method, but it's not a big deal. Also, we don't (yet?) do this in other models. This way, each model could take a little more responsibility for its own representation, and the description generator can handle the formatting external to the model? I could go either way on it. Would like to hear what you think, though.